portal/views/assessment_engine.py

```diff
@@ -52,10 +53,11 @@
 @assessment_engine_api.route(
     '/patient/<int:patient_id>/assessment',
     defaults={'instrument_id': None},
-)
+    methods=('OPTIONS', 'GET'))
 @assessment_engine_api.route(
-    '/patient/<int:patient_id>/assessment/<string:instrument_id>'
-)
+    '/patient/<int:patient_id>/assessment/<string:instrument_id>',
+    methods=('OPTIONS', 'GET'))
+@crossdomain()
```

Comment:
Did you try passing [`automatic_options=True`](https://github.com/uwcirg/true_nth_usa_portal/blob/develop/portal/views/crossdomain.py#L32)? It might obviate the need to add `OPTIONS` to all the `methods` arguments to `route()`.

[Good reference implementation here](https://github.com/corydolphin/flask-cors/blob/master/flask_cors/decorator.py#L106-L119)

Reply:
I didn't, but I will. This will make the change a heck of a lot simpler.